src/content/en/2020/resource-hints.md

```diff
@@ -329,57 +307,60 @@ As mentioned earlier, the [PRPL Pattern](https://addyosmani.com/blog/the-prpl-pa
 
-With `preload` and `prefetch`, the priority is set by the browser depending on the type of resource. By using Priority Hints we can force the browser to change the default option.
+{# TODO(editors): Should we keep this shorter to prevent small amounts of scrolling? #}
+```html
+<!-- We want to initiate an early fetch for a resource, but also deprioritize it -->
+<link rel="preload" href="/js/script.js" as="script" importance="low">
 
-{# TODO - simplify all these figures #}
+<!-- An image the browser assigns "High" priority, but we don't actually want that. -->
+<img src="/img/in_view_but_not_important.svg" importance="low" alt="I'm not important!">
+```
+
+With `preload` and `prefetch`, the priority is set by the browser depending on the type of resource. By using Priority Hints we can force the browser to change the default option.
 
 {{ figure_markup(
-  caption="The rate of priority hint adoption on mobile",
+  caption="The percent of Priority Hint adoption on mobile.",
   content="0.77%",
   classes="big-number",
   sheets_gid="1596669035",
   sql_file="priority_hints.sql"
 ) }}
 
-So far only 0.77% websites adopted this new hint as Chrome is still [actively](https://www.chromestatus.com/features/5273474901737472) experimenting, and at the time of this article's release the feature is on-hold.
-
-{{ figure_markup(
-  caption="Priority hints on mobile are on script elements",
-  content="80%",
-  classes="big-number",
-  sheets_gid="800402946",
-  sql_file="priority_hints_by_element.sql"
-) }}
+So far only 0.77% websites adopted this new hint as Chrome is still [actively](https://www.chromestatus.com/features/5273474901737472) experimenting, and at the time of this article's release the feature is on-hold. 
 
 The largest use is with script elements, which is unsurprising as the number of JS primary and third-party files continues to grow.
 
+{# TODO(authors): Make sure the query link is correct, and that the gid is correct. #}
 {{ figure_markup(
-  caption="Priority hints on mobile have \"low\" importance",
-  content="16.11%",
+  caption="The percent of mobile resources with a hint that use the \"low\" priority.",
+  content="16%",
   classes="big-number",
   sheets_gid="1098063134",
-  sql_file="priority_hints_by_element_and_importance.sql"
+  sql_file="priority_hints_by_importance.sql"
 ) }}
 
-There are over 79% of resources with "high" priority, but something we should pay even more attention to is the 16% of resources with "low" priority. Priority Hints have a clear advantage as a defense mechanism rather than an offense, by helping the browser decide what to de-prioritize and giving back significant CPU and Bandwidth to complete critical requests first.
+{# TODO(authors): Review this revised section. #}
+{# TODO(authors): 79 or 83? 79% is from gid=1663326536 for mobile + script + high, while 83% is from gid=1098063134 and corresponds to the 16% "low" we use in the figure. #} 
```

Comment:
@Zizzamia @khempenius I was unsure which number to refer to here. The 16% appears to come from [here](https://docs.google.com/spreadsheets/d/1lXjd8ogB7kYfG09eUdGYXUlrMjs4mq1Z7nNldQnvkVA/edit#gid=1098063134&range=E5), but the (original) 79% comes from [here](https://docs.google.com/spreadsheets/d/1lXjd8ogB7kYfG09eUdGYXUlrMjs4mq1Z7nNldQnvkVA/edit#gid=1663326536&range=F2), while the real value should be from [here](https://docs.google.com/spreadsheets/d/1lXjd8ogB7kYfG09eUdGYXUlrMjs4mq1Z7nNldQnvkVA/edit#gid=1098063134&range=E2) as far as I can tell.

Reply:
You right, great catch!
Because the global adoption of Priority Hints is less than 1%, I figured was not worth to make any split between tag type and just keep it simple and focus more on the importance (low vs high).

Yeah 83% is the right pick!!! In some way I was thinking something but I wrote something else LOOL.